InstantReview/App.xaml.cs

```diff
@@ -33,6 +33,7 @@ public App(ContainerBuilder containerBuilder)
             navigationPage.PushAsync(CreateMainPage());
             masterDetailPage = CreateMasterDetailPage(navigationPage);
             MainPage = masterDetailPage;
+            MainPage = new ThankYouPage();
```

Comment:
Why set thank you page as main page? Doesn't make any sense.

Reply:
Just to see it when the app ran. Obviously it is not going to be the main page.

Reply:
Well can you change it so that it doesn't conflict with the current develop branch? I can't accept something like this. Same applies for the other PR.

Reply:
OK